Library/Formula/dspdfviewer.rb

```diff
@@ -0,0 +1,23 @@
+class Dspdfviewer < Formula
+  desc "Dual-Screen PDF Viewer for latex-beamer"
+  homepage "http://dspdfviewer.danny-edel.de"
+  url "https://github.com/dannyedel/dspdfviewer/archive/v1.13.1.tar.gz"
+  sha256 "333588de0316cfdb5821b8484ee55690dfa3c7431b67c126bfdbe9c9cc3e1ed4"
+
+  head "https://github.com/dannyedel/dspdfviewer.git"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on "poppler" => "with-qt"
```

Comment:
afaik the support is there in the master branch but not in the latest release. So if it's stable we should be able to switch to qt5